### PR TITLE
fix #1543 star interface

### LINKED ISSUE #1543: Stars interface. [bug][good first issue]
# Bug Report

## Steps to Reproduce

Click on an item with "X Stars"

## Expected Behavior

Preload the stars field with the amount of stars currently selected.

## Actual Behavior

Displays 0 stars selected by default.

## Other Context & Screenshots

Changing the stars does work, it just doesn't load the right default in. :sob: 

Selection_063
Selection_062

## Technical Details

- Directus App: 7.0.3

### PR DESCRIPTION
**Pull request by dev7ch**
closes #1543

### CODE REVIEW

File: src/interfaces/rating/input.vue [hunk 1]
```diff
@@ -18,7 +18,9 @@
         :value="String(value) || '0'"
         @input="updateValue"
       ></v-input>
-      <span>/ {{ options.max_stars }}</span>
+      <span v-html="(!!value ? String(value) : '0') + '/' + options.max_stars">
```
Comment (rijkvanzanten) [inline src/interfaces/rating/input.vue]: Does this have to be the unsafe raw html? Why not render this as escaped text in the span?

File: src/interfaces/rating/input.vue [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   <div class="rating">
     <stars
-      v-if="options.display == 'star'"
+      v-if="options.display === 'star'"
       :options="options"
-      :rating.sync="rating"
+      :rating.sync="!!value ? value : rating"
       :readonly="readonly"
-      @update:rating="updateValue"
+      v-on:update:rating="updateValue"
```
Comment (rijkvanzanten) [inline src/interfaces/rating/input.vue]: Isn't `@` the shorthand for `v-on`?

Comment (shealavington) [reply src/interfaces/rating/input.vue]: I thought so which is why I used the @ there for the last fix I did. 🤔

Comment (dev7ch) [reply src/interfaces/rating/input.vue]: upps yes

File: src/interfaces/rating/input.vue (hunk 1 above)
Comment (dev7ch) [reply src/interfaces/rating/input.vue]: ok done

**Pull request merged**